backend/src/core/models/game.js

```diff
@@ -11,17 +11,32 @@ export class Game {
   imageUrl;
   playerHistory;
 
+  copy() {
+    const copy = new Game();
+    copy.id = this.id;
+    copy.name = this.name;
+    copy.releaseDate = this.releaseDate;
+    copy.developers = this.developers.slice();
+    copy.genres = this.genres.slice();
+    copy.description = this.description;
+    copy.imageUrl = this.imageUrl;
+    copy.playerHistory = PlayerHistory.manyFromDbEntry(this.playerHistory);
+
+    return copy;
+  }
+
   // prettier-ignore
-  static fromSteamApp(steamApp, releaseDate, developers, genres, description) {
+  static fromSteamApp(steamApp, page) {
     const game         = new Game();
     game.id            = steamApp.appid;
     game.name          = steamApp.name;
-    game.releaseDate   = releaseDate;
-    game.developers    = developers;
-    game.genres        = genres;
-    game.description   = description;
+    game.releaseDate   = game.getReleaseDate(page);
```

Comment:
Ok, now it gets a bit more involved and complicated and because you will change a lot I don't want to go to deep into it but I will say a few things for now and we talk about them if its unclear.

You are using those methods to set some values on the model but are not used by any external client, so they should be private OR someone in the future will get the idea to use them somewhere :)...also a better name would be: `extractReleaseDateFrom(...) OR parseReleaseDateFrom(...)`. Same for the other 3 methods. You can make them private static or create functions in this function.